CruzHacks/cruzhacks-2021-website
# Team Page

Problem
=======
As an organization 
I want to see a team page 
So that we can credit the people helping organize this event

[Link to Asana Ticket](https://app.asana.com/0/1197738625416321/1199387105632111/f)

Solution
========
I created 3 components in order to create the Team page view. I created the `Hero`, `Card`, and `Footer` components, which all make up the `Team` page.

Change Summary:
------------------
* added imaged and svgs d28efc9
* added cards component and props 98f9a30
* added hero and footer components to index page 65eac24
* added mixins to styling 7cb8dcb
* added teams endpoint to routes 55881ba
* added ez-index to navbar … 768f2e8
* added styled backdrop d53ecff

Steps to Verify:
----------------
1. Checkout this branch and start a local development environment 
2. Navigate to the `/team` endpoint 
3. Verify that: 
* webpage is responsive to all screen sizes and that all visible components from figma design are present 
* Arrow at the bottom takes you to the top of the page seamlessly
* LinkedIn links take you to correct profile   

Screenshots:
-----------------------
![Hnet-image (1)](https://user-images.githubusercontent.com/40477553/104153181-85cdf800-5396-11eb-84e3-c6259547172b.gif)
